frontend/src/components/GlobalStateProvider.tsx

```diff
@@ -20,27 +20,33 @@ export function getExistingSettings(): NodeManagerSettingStrings {
     const network = localStorage.getItem('MUTINY_SETTINGS_network') || process.env.REACT_APP_NETWORK;
     const proxy = localStorage.getItem('MUTINY_SETTINGS_proxy') || process.env.REACT_APP_PROXY;
     const esplora = localStorage.getItem('MUTINY_SETTINGS_esplora') || process.env.REACT_APP_ESPLORA;
+    const rgs = localStorage.getItem('MUTINY_SETTINGS_rgs') || process.env.REACT_APP_RGS || "";
+    const lsp = localStorage.getItem('MUTINY_SETTINGS_lsp') || process.env.REACT_APP_LSP || "";
```

Comment:
should add `REACT_APP_RGS` and `REACT_APP_LSP` to the .env.sample file as well

Reply:
added